### PR TITLE
Workingbranch3

### PR DESCRIPTION
**Pull request by jennymegan**
Final draft - hoping ready to merge from from wb3 into portfolio.

### CODE REVIEW

File: indexstyle.css [hunk 1]
```diff
@@ -117,6 +119,8 @@ li a:hover {
 .intropage .container {
 	height: 100vh;
 	align-items: center;
+	background-image: url(handshake.jpg);
```
Comment (JoshThomasField) [inline indexstyle.css:152]: Consider changing this background in your media queries, on a phone it looks a bit strange.

File: index.html [hunk 2]
```diff
@@ -77,14 +74,18 @@ <h4>January 2021, Bristol UK</h4>
...
+				<img class="aboutmeimg" src="jennyaboutme.jpg" alt="Jenny Allton-Twist looking happy"/>
 			</div>
 			<div class="aboutmebox">
-				<h2>About Me</h2>
-				<p>Some stuff about me, a paragraph in which I talk humorously and succintly about my many achievements and accolades, as well as the interesting and altruistic activities I undertake in my spare time.</p>
+				<h2>ABOUT ME</h2>
+				<p>I’m a full-stack software developer with an interest in back-end development and machine learning. I graduated from<a href="https://mayden.academy/"> Mayden Academy</a> in December 2020.</p>
+				<p>Prior to my formal training I have almost a decade’s experience in project management and product design in a superyacht interior design company. My strengths are in my problem-solving and communication skills, both written and verbal.</p>
+				<p>I love running, reading trashy urban fiction, and learning languages that are spoken by people as well as languages used by computers. Oh, and I really like burritos.</p>
+				<p></p>
```
Comment (JoshThomasField) [inline index.html:86]: Is this extra paragraph tag intentional?

File: indexstyle.css [hunk 3]
```diff
@@ -233,6 +250,17 @@ footer {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (JoshThomasField) [inline indexstyle.css:285]: Not sure this many spaces is necessary?

File: project.html [hunk 4]
```diff
@@ -5,34 +5,39 @@
...
 		 </ul>
-		 <!-- <div><img src="" alt=""/> Place for logo -->
 	</nav>
 
-	<div class="container project">
-		<div class="projectimg"><img src="projectimg.jpg" alt="Photos of Project" width="600"></div>
-		<div> 
-			<h2>Project The First</h2>
-			<p>This went well and we were all thrilled.</p>
+div class="aboutme" id="aboutme">
```
Comment (JoshThomasField) [inline project.html]: Have you missed an opening tag here? <

File: indexstyle.css [hunk 5]
```diff
@@ -233,6 +250,17 @@ footer {
 }
```
Comment (JoshThomasField) [inline indexstyle.css:277]: I'd probably increase the size of your social Icons a little, they're kinda small at the minute.

**Review by JoshThomasField — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (jennymegan) [reply index.html:86]: Yup, it was to give space before the final p.

File: project.html (hunk 4 above)
Comment (jennymegan) [reply project.html]: Yup!

File: indexstyle.css (hunk 1 above)
Comment (jennymegan) [reply indexstyle.css:152]: agreed. done

File: indexstyle.css (hunk 5 above)
Comment (jennymegan) [reply indexstyle.css:277]: Done! I was using an old stylesheet. Sorted now.

**Review by JoshThomasField — APPROVED** [review]

**Pull request merged**